Library/Homebrew/test/testing_env.rb

```diff
@@ -57,9 +57,7 @@ def after_teardown
       super
       files_after_test = []
       Find.find(TEST_TMPDIR) { |f| files_after_test << f.sub(TEST_TMPDIR, "") }
-      if @__files_before_test != files_after_test
-        @@log.puts location, diff(@__files_before_test, files_after_test)
-      end
+      @@log.puts location, diff(@__files_before_test, files_after_test) if @__files_before_test != files_after_test
```

Comment:
Better to try and keep this below 80 chars by e.g. splitting this into an`if .. end`.